index.txt

```diff
@@ -4,7 +4,7 @@
 #
 #   Homepage: https://caddyserver.com
 #   Issues:   https://github.com/caddyserver/getcaddy.com/issues
-#   Requires: bash, curl or wget, tar or unzip
+#   Requires: bash, coreutils, curl or wget, tar or unzip
```

Comment:
It depends a bit on the packaging and what things are called of course. On my machine, `rm`, `type` and `mv` are part of coreutils.

Reviewing all this, `setcap` is not part of coreutils, so should be added separately. (On my system, it's part of `libcap2-bin` but I don't know how universal that is...)

I would propose to switch to just mentioning the names of all the external binaries that are required: `curl`/`wget`, `rm`, `cp`, `type`, `setcap` (not required, but will be used when present)